pkg/quota/image/imagestream_evaluator.go

```diff
@@ -25,50 +25,53 @@ const (
 // NewImageStreamEvaluator computes resource usage of ImageStreams. Instantiating this is necessary for
 // resource quota admission controller to properly work on image stream related objects.
 func NewImageStreamEvaluator(osClient osclient.Interface) kquota.Evaluator {
-	computeResources := []kapi.ResourceName{
-		imageapi.ResourceImages,
-	}
-
-	matchesScopeFunc := func(kapi.ResourceQuotaScope, runtime.Object) bool { return true }
-	getFuncByNamespace := func(namespace, name string) (runtime.Object, error) {
-		return osClient.ImageStreams(namespace).Get(name)
-	}
 	listFuncByNamespace := func(namespace string, options kapi.ListOptions) (runtime.Object, error) {
 		return osClient.ImageStreams(namespace).List(options)
 	}
 
+	evaluator := NewImageStreamAdmissionEvaluator(osClient)
+	genericEvaluator := evaluator.(*generic.GenericEvaluator)
+	genericEvaluator.Name = imageStreamEvaluatorName
+	genericEvaluator.InternalOperationResources = nil
```

Comment:
I thought this was used to match which resources to call out on.  Does `nil` here mean something special?